src/AzureExtension/Providers/RepositoryProvider.cs

```diff
@@ -347,96 +359,138 @@ public void Dispose()
         GC.SuppressFinalize(this);
     }
 
-#pragma warning disable CA1309 // Use ordinal string comparison. I want to use linguistic comparison.
-    public IAsyncOperation<RepositoriesResult> GetRepositoriesAsync(IReadOnlyDictionary<string, string> fieldValues, IDeveloperId developerId)
+    public IAsyncOperation<RepositoriesSearchResult> GetRepositoriesAsync(IReadOnlyDictionary<string, string> fieldValues, IDeveloperId developerId)
     {
-        // First query will find repos only if the user made a PR.
-        var shouldIgnorePRDateOld = _shouldIgnorePRDate;
-        _shouldIgnorePRDate = true;
         return Task.Run(() =>
         {
             try
             {
                 var serverToUse = fieldValues.ContainsKey(_server) ? fieldValues[_server] : string.Empty;
+                _serverToSearch = serverToUse;
+
                 var organizationToUse = fieldValues.ContainsKey(_organization) ? fieldValues[_organization] : string.Empty;
+                _organizationToSearch = organizationToUse;
+

```

Comment:
could we do `_serverToSearch = fieldValues.ContainsKey(_server)...` and `_organizationToSearch = fieldValues.ContainsKey(_organization)...`

Reply:
Changed.